Data/ApplicationDbContext.cs

```diff
@@ -57,6 +58,11 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             {
                 entity.ToTable(name: "Image");
             });
+
```

Comment:
Jak nie chcesz dodawać EntityTypeConfiguration to tutaj podefiniuj np Tabela Training - property np Name  - HasMaxLength(200)